src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1388,47 +1388,51 @@ private static class AutoLoadCandidatesService extends Service<Void> {
     @Override
     protected Task<Void> createTask() {
       Task<Void> task =
-              new Task<>() {
-                @Override
-                protected Void call() {
-                  // Gather unloaded names from each of the sources and place into the HashSet
-                  Set<String> unloadedNames = new HashSet<>();
-                  for (CvrSource source : sources) {
-                    Provider provider = ContestConfig.getProvider(source);
-                    try {
-                      List<CastVoteRecord> castVoteRecords = new ArrayList<>();
-                      BaseCvrReader reader = provider.constructReader(config, source);
-                      reader.readCastVoteRecords(castVoteRecords, new HashSet<>());
-                      unloadedNames.addAll(
-                          reader.gatherUnknownCandidates(castVoteRecords).keySet());
-                    } catch (ContestConfig.UnrecognizedProviderException e) {
-                      Logger.severe(
-                          "Unrecognized provider \"%s\" in source file \"%s\": %s",
-                          source.getProvider(), source.getFilePath(), e.getMessage());
-                    } catch (CastVoteRecord.CvrParseException | IOException e) {
-                      Logger.severe("Failed to read source file \"%s\": ",
-                          source.getFilePath(), e.getMessage());
-                    }
-                  }
-
-                  // Validate each name and add to the table of candidates
-                  int successCount = 0;
-                  for (String name : unloadedNames) {
-                    Candidate candidate = new Candidate(name, null, false);
-                    Set<ValidationError> validationErrors =
-                            ContestConfig.performBasicCandidateValidation(candidate);
-                    if (validationErrors.isEmpty()) {
-                      tableViewCandidates.getItems().add(candidate);
-                      successCount++;
-                    } else {
-                      Logger.warning("Candidate \"%s\" failed to load.", name);
-                    }
-                  }
-
-                  Logger.info("Auto-loaded %d candidates.", successCount);
-                  return null;
+          new Task<>() {
+            @Override
+            protected Void call() {
+              Logger.info("Auto-loading candidates from CVR files...");
+              if (sources.isEmpty()) {
+                Logger.warning("No CVR files specified!");
+                return null;
```

Comment:
Is there an issue with multiple returns here @HEdingfield?

Reply:
Yeah, this isn't allowed. Also line 1433 below.

Reply:
Good catch. Soooo dumb. I'll clean this up. 1433 is necessary though, because we're overriding the `call` method so it needs some kind of return.

Reply:
Oops, right. Hard to read indentation when viewing in GitHub.

Reply:
Filed #693 for this.